frontend/src/stores/courses.js

```diff
@@ -25,22 +25,23 @@ const isAvailable = (
     reservedCourseList
 ) => {
     const openedYet = isOpenYet(courseItem);
-    const closedYet = isClosedYet(courseItem);
+    const closedYet = !isClosedYet(courseItem);
```

Comment:
Quite misleading to assign opposite of `isClosedYet` to variable called `closedYet`.